src/components/Sidebar.tsx

```diff
@@ -248,23 +260,9 @@ const TabIcon = styled.img`
 // Network switch
 
 const NetworkMenu = styled(Menu)`
-  position: absolute !important;
-  bottom: 0;
-  right: 0;
-  left: 0;
-`
-
-const AlephiumLogoMainnet = styled(AlephiumLogo)`
-  path {
-    stroke-width: 18 !important;
-  }
-`
-
-const AlephiumLogoTestnet = styled(AlephiumLogo)`
-  path {
-    stroke-width: 18 !important;
-    stroke: ${({ theme }) => theme.textSecondary} !important;
-  }
+  border-width: 1px 0;
+  border-style: solid;
+  border-color: ${({ theme }) => theme.borderSecondary};
 `
 
 export default Sidebar
```

Comment:
Oh sorry, I missed your updates!

I would still make the following changes to this file so that we don't define a new anonymous function everytime we want to close the sidebar:

```diff
diff --git a/src/components/Sidebar.tsx b/src/components/Sidebar.tsx
index 164cce7..2a9aea1 100644
--- a/src/components/Sidebar.tsx
+++ b/src/components/Sidebar.tsx
@@ -14,7 +14,7 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the library. If not, see <http://www.gnu.org/licenses/>.
 
-import { useContext, useEffect, useRef } from 'react'
+import { useContext, useEffect, useRef, useCallback } from 'react'
 import { Link, NavLink } from 'react-router-dom'
 import styled, { useTheme } from 'styled-components'
 
@@ -49,6 +49,10 @@ const Sidebar = ({ sidebarState }: { sidebarState: SidebarState }) => {
     }
   }
 
+  const closeSidebar = useCallback(() => {
+    setSidebarState('close')
+  }, [setSidebarState])
+
   useEffect(() => {
     if (windowWidth) {
       if (
@@ -57,24 +61,24 @@ const Sidebar = ({ sidebarState }: { sidebarState: SidebarState }) => {
         windowWidth < deviceSizes.tablet &&
         open
       ) {
-        setSidebarState('close')
+        closeSidebar()
       }
 
       lastWindowWidth.current = windowWidth
     }
-  }, [setSidebarState, windowWidth])
+  }, [closeSidebar, windowWidth])
 
   return (
     <>
       <SidebarContainer open={sidebarState === 'open'}>
-        <CloseButton onClick={() => setSidebarState('close')}>{<X />}</CloseButton>
+        <CloseButton onClick={closeSidebar}>{<X />}</CloseButton>
         <Header>
           <Link to="/">
             <Logo alt="alephium" src={theme.name === 'light' ? logoLight : logoDark} />
           </Link>
           <NetworkMenu
             label={isMainnet ? 'Mainnet' : 'Testnet'}
-            icon={isMainnet ? <NetworkLogo network="mainnet" /> : <NetworkLogo network="testnet" />}
+            icon={<NetworkLogo network={networkType || 'testnet'} />}
             items={[
               {
                 text: 'Mainnet',
@@ -93,13 +97,13 @@ const Sidebar = ({ sidebarState }: { sidebarState: SidebarState }) => {
         <Navigation>
           <NavigationTitle>MENU</NavigationTitle>
           <Tabs>
-            <Tab to="/blocks" onClick={() => setSidebarState('close')}>
+            <Tab to="/blocks" onClick={closeSidebar}>
               <TabIcon src={blockIcon} alt="blocks" /> Blocks
             </Tab>
-            <Tab to="/addresses" onClick={() => setSidebarState('close')}>
+            <Tab to="/addresses" onClick={closeSidebar}>
               <TabIcon src={addressIcon} alt="addresses" /> Addresses
             </Tab>
-            <Tab to="/transactions" onClick={() => setSidebarState('close')}>
+            <Tab to="/transactions" onClick={closeSidebar}>
               <TabIcon src={transactionIcon} alt="transactions" /> Transactions
             </Tab>
           </Tabs>
@@ -109,7 +113,7 @@ const Sidebar = ({ sidebarState }: { sidebarState: SidebarState }) => {
       <AnimatePresence>
         {sidebarState === 'open' && (
           <Backdrop
-            onClick={() => setSidebarState('close')}
+            onClick={closeSidebar}
             animate={{ opacity: 0.4 }}
             exit={{ opacity: 0 }}
             initial={{ opacity: 0 }}
```

Reply:
Great remark. Done.